internal/cmd/root.go

```diff
@@ -122,6 +124,36 @@ crush run -q "Generate a README for this project"
 	},
 }
 
+var updateCmd = &cobra.Command{
+	Use:   "update",
+	Short: "Check for updates",
+	Long:  `Check if a new version of crush is available.`,
+	Example: `
+# Check for updates
+crush update
```

Comment:
originally it was crush update because it would update the version, but now is just a version check